source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -28,8 +28,16 @@ Use the :mc:`mc support perf` command to review the performance of the S3 API (r
 .. end-mc-support-perf-desc
 
 The resulting tests can provide general guidance of deployment performance under S3 ``GET`` and ``PUT`` requests and identify any potential bottlenecks.
+
+.. admonition:: Other S3 API calls suspended during testing
+   :class: note
+
+   For S3 testing, :mc:`mc support perf` temporarily suspends S3 API calls during a test.
+   They are automatically restarted after the test is complete or if the command is cancelled.
+
```

Comment:
@klauspost which tests do this? `mc support perf client`? What exactly happens during the test? Requests rejected?

Reply:
Also want to understand what "freeze" means. Maybe "suspend" isn't the right word here either. (And the other command is named `unfreeze`.) It would be better to give some additional context.

Reply:
I think this also applies to `mc support perf net`

Reply:
Requests are held back in and queued.